src/AppInstallerCLICore/Workflows/RepairFlow.cpp

```diff
@@ -151,6 +169,15 @@ namespace AppInstaller::CLI::Workflow
         // Outputs:None
         void ApplicabilityCheckForAvailablePackage(Execution::Context& context)
         {
+            InstallerTypeEnum installedPackageInstallerType = GetInstalledPackageInstallerType(context);

```

Comment:
if (!context.contains(Execution::data::installer))
{
return;
}

and no need to check installed type below